pkg/proto/sfu.proto

```diff
@@ -42,21 +42,15 @@ message SubscribeReply {
 }
 
 message Connect {
-    SessionDescription description = 1;
+    bytes sdp = 1;
     Options options = 2;
 }
 
 message Trickle {
     string candidate = 1;
 }
 
-message SessionDescription {
-    string type = 1; // "answer" | "offer" | "pranswer" | "rollback"
-    string sdp = 2;
-}
-
 message Options {
     uint32 bandwidth = 1;
     bool transportcc = 2;
-    string codec = 3;
```

Comment:
Instead of explicitly providing a codec like this, we use the sdp codec ordering for priority. i'll provide a utility in the ion-sdk-js to specify preferred codec